math.c

```diff
@@ -179,14 +179,20 @@ math_tan(VALUE unused_obj, VALUE x)
  */
 
 static VALUE
-math_acos(VALUE unused_obj, VALUE x)
+math_acos_or_asin(VALUE x, double func(double))
 {
     double d;
 
     d = Get_Double(x);
     /* check for domain error */
     if (d < -1.0 || 1.0 < d) domain_error("acos");
```

Comment:
The argument of ``domain_error`` is different for acos and asin.

Reply:
Thanks! I'll fixed it.